src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -73,7 +73,7 @@ public Number get(final Player player) {
 	@Nullable
 	@Override
 	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
-		return (mode != ChangeMode.REMOVE_ALL) ? CollectionUtils.array(Number.class) : null;
+		return (mode != ChangeMode.REMOVE_ALL && mode != ChangeMode.TOGGLE) ? CollectionUtils.array(Number.class) : null;
```

Comment:
You should use a switch statement here (like in other spots)

Reply:
oh no
as I said the TOGGLE changer is the only one I didn't look into 😬

Reply:
Oh, right haha. Might as well leave these here anyway >:)